guide/xml/portgroup-python.xml

```diff
@@ -181,7 +181,7 @@
         <listitem>
           <para>The Python dynamic library path, i.e.,
           <filename>${python.prefix}/Python</filename> (framework builds) or
-          <filename>${prefix}/lib/libpython2.4.dylib</filename> (python24).</para>
+          <filename>${prefix}/lib/libpython3.6.dylib</filename> (python36).</para>
```

Comment:
This is incorrect; python36 does not install this file. The reference to python24 was quite deliberate, but since that port has now been removed, all remaining python ports are framework builds and this exception is no longer relevant.